src/messages/index.js

```diff
@@ -1,25 +1,25 @@
 import "./messages.css";
 
-const Messages = () => (
-  <div class="messages">
-    <div id="streak-message" class="show">
-      Current Streak: <span id="streak">@Model.Streak</span>
-    </div>
-    <div id="attempts-message">
-      Attempts Remaining Before Streak Lost: <span id="attempts-left">3</span>
-    </div>
-    <div id="incorrect-display">
-      <div class="incorrect-message">
-        Incorrect!
-        <span id="incorrect-message-reset">Streak Reset.</span>
-      </div>
-      <div class="incorrect-answer">
-        <a href="#answer" id="incorrect-answer-link">
-          Show Answer (Lose Streak)
-        </a>
-      </div>
-    </div>
-  </div>
-);
+const Messages = (props) => {
+  let visible = false;
+  let isCorrect = false;
```

Comment:
I would move these up and make them properties of the component instead. Right now your component just accepts `checkWin` which means it needs to know what the possible values of the enum are. Instead it should just know the values of two booleans. It's the parent component that should be responsible for determining what the value of those booleans are (in other words, move this logic up to the Quiz)